src/vochain/vochain.proto

```diff
@@ -96,6 +98,18 @@ message ProofArbo {
         bytes siblings = 2;
 }
 
+// Groth16 zkSNARK proof + public inputs
+message ProofZkSNARK {
+        // a represents a G1 point
+        repeated string a = 1;
```

Comment:
Why strings instead of bytes?

Reply:
The reasoning has been:
- client side circomlib works with bigints/strings, and at the end generates a json file
- the types in the go side are not even byte arrays, are `bn256.G1 & G2` points
There exist methods to import Proof from string data structure, so thought that would make things easier by using strings.
But, I'm open to any other approach.

Reply:
Isn't it better (and more elegant) to use `big.Int.Bytes()`?

Reply:
We can go with `bytes` but then we need to define which endianness we use, and which padding/length between the different libraries of the different languages (js, go, dart, ...). Like, for example, is not the same little-endian or big-endian representation, or maybe a lib adds padding of zeroes until a length of 32bytes, and the other one no.
Also, the outputs and inputs in circom are using string representation of bigints, and this kind of data packets are thought to be used when working with circom.

Reply:
I'm not sure. On my experience hexadecimal big.Int representation is quite standard between languages (we are doing it with Go, JS and DART), a []bytes representation is directly the hexa reprsentation (no paddings or litle/big endian problems).

Do you have an example of a language implementing bigInt hexa representation differently?

Anyway there is not a big difference from using strings (just some bytes more required), so I'd let it to your decision.